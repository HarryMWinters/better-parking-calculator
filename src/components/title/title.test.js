import React from "react";
import renderer from "react-test-renderer";
import Title from "./title";

it("Matches snapshot", () => {
  const tree = renderer.create(<Title />).toJSON();
  expect(tree).toMatchSnapshot();
});
