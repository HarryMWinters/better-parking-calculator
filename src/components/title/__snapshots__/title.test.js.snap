// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches snapshot 1`] = `
<div
  className="jumbotron"
>
  <h1>
    A Better Parking Calculator
  </h1>
  <p>
    This project was inspired by an interview challenge to "find the bugs" in a web app that calculates parking costs.
    <a
      href="http://adam.goucher.ca/parkcalc/"
      rel="noopener noreferrer"
      target="_blank"
    >
       
      Original app
    </a>
     
    (frequently down)
    . Background info
     
    <a
      href="https://www.infoq.com/news/2010/05/testing_challenge/"
      rel="noopener noreferrer"
      target="_blank"
    >
      here
    </a>
    .
  </p>
  <p>
    If you're super impressed by bootstrap themes, Fullstack mess-arounds and pictures of software engineers in the high Sierra check out my
     
    <a
      href="https://github.com/HarryMWinters"
      rel="noopener noreferrer"
      target="_blank"
    >
      github
    </a>
     
    and
     
    <a
      href="https://www.linkedin.com/in/code-bio/"
      rel="noopener noreferrer"
      target="_blank"
    >
      LinkedIn
    </a>
    .
  </p>
</div>
`;
